### PR TITLE
Add vlang extension

### PR DESCRIPTION
**Pull request by daaniiieel**
This PR adds the official V programming language vscode extension.

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "extensions": [
+    {
+      "id": "0x9ef.vscode-vlang",
+      "repository": "https://github.com/vlang/vscode-vlang",
```
Comment (jankeromnes) [inline extensions.json:5]: Oh nice, the extension actually provides a `.vsix` release package! In that case I'd prefer to upload that directly, as re-packaging ourselves carries the risk of doing it the wrong way and accidentally uploading a dysfunctional extension.

```suggestion
      "download": "https://github.com/vlang/vscode-vlang/releases/download/0.1.12/vscode-vlang-0.1.2.vsix",
```

**Review by jankeromnes — APPROVED** [review]
Many thanks for the contribution! 🎉

Approved with one in-line suggestion (that I'll be able to apply myself before merging 😇 ).

**Comment by daaniiieel** [comment]
Thank you!

**Comment by jankeromnes** [comment]
Looks like auto-publishing was successful! 🎉https://open-vsx.org/extension/0x9ef/vscode-vlang

**Pull request merged**